fluent/fluent.go

```diff
@@ -408,6 +408,15 @@ type gRIBIModify struct {
 	parent *GRIBIClient
 }
 
+// InjectRequest injects a gRIBI ModifyRequest that is created by an external
+// entity into the modify stream. No validation of the input message is performed.
+// It is intended to allow for invalid messages that the fluent library does not
+// allow the creation of to be sent to a server.
+func (g *gRIBIModify) InjectRequest(t testing.TB, m *spb.ModifyRequest) *gRIBIModify {
```

Comment:
reminds me again, why we want to return `*gRIBIModify`?

Reply:
This allows us to chain calls - so you can do `(*gRIBIModify).InjectRequest(...).AddEntry(...)`. In these cases it seems a little less likely that a caller would actually do this, but this is the fluent API pattern.